queue_services/account-mailer/src/account_mailer/email_processors/common_mailer.py

```diff
@@ -44,6 +44,12 @@ def process(org_id, recipients, template_name, subject, logo_url, **kwargs) -> d
         'logo_url': logo_url,
         **kwargs
     }
+
+    # if bceid resubmit request, get remarks length and pass along in template
+    if 'remarks' in jinja_kwargs:
```

Comment:
can we pass this from auth-api and avoid the logic here?
or something sort of that?

Reply:
Refactored accordingly